src/js/extensions/anchor-preview.js

```diff
@@ -20,11 +20,9 @@ var AnchorPreview;
         previewValueSelector: 'a',
 
         /* ----- internal options needed from base ----- */
-        'window': window,
-        'document': document,
```

Comment:
Yes indeed.  They're passed from the core editor's `ownerDocument` and `contentWindow` options.